core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -520,7 +520,7 @@ private[deploy] class SparkSubmitArguments(args: Seq[String], env: Map[String, S
         |                              (Default: client).
         |  --class CLASS_NAME          Your application's main class (for Java / Scala apps).
         |  --name NAME                 A name of your application.
-        |  --jars JARS                 Comma-separated list of local jars to include on the driver
```

Comment:
This is a new feature(support non-local jars), shall we create a separated jira ticket?